themes/default/content/docs/guides/crosswalk/kubernetes/app-services.md

```diff
@@ -16,41 +16,27 @@ These services tend to include datastores, and managers for ingress, DNS, and TL
 They can be shared amongst several apps or be specific to workloads, and are
 usually a mix of cloud provider and custom services.
 
-{{% choosable cloud aws %}}
+## Example Code
 
-The full code for the AWS app services stack is on [GitHub][gh-repo-stack].
+{{% choosable cloud aws %}}
 
-<!-- markdownlint-disable url -->
-[gh-repo-stack]: https://github.com/pulumi/kubernetes-guides/tree/master/aws/05-app-services
```

Comment:
i didnt really understand the point of these since they arent being used anywhere else on the page, but if someone has more context pls let me know

Reply:
Ah, nice catch -- this is a bug resulting from how these links end up being rendered at build-time (because the name `gh-repo-stack` is being reused). I generally don't love the notation that was used here, so I'm definitely 👍 on the changes to use normal Markdown links.

I'm not sure I'd add the Example Code heading at the top like this, though, since this page is meant as more of a guide than a reference like the API docs pages (assuming that's what you were thinking by adding it here). Let me read through it, though, and I'll chime back in after the break.

However I don't think I'd change the overall hierarchy here.

Reply:
cool, yah i only added that because i couldn't come up with good enough link text that didn't require some kind of header on that section; tots open to other ideas

Reply:
@cnunciato any thoughts on how to make this better? since the bad links were user reported id like to get them fixed soon.

Reply:
Sure, if you want to get it fixed now, you can just change the format of these three links to be regular Markdown expressions, as that's what's causing the problem. So for example, for each cloud, from this:

```
{{% choosable cloud aws %}}

The full code for the AWS app services stack is on [GitHub][gh-repo-stack].

<!-- markdownlint-disable url -->
[gh-repo-stack]: https://github.com/pulumi/kubernetes-guides/tree/master/aws/05-app-services
<!-- markdownlint-enable url -->

{{% /choosable %}}
```

to this:

```
{{% choosable cloud aws %}}

The full code for the AWS app services stack is on [GitHub][https://github.com/pulumi/kubernetes-guides/tree/master/aws/05-app-services].

{{% /choosable %}}
```

That'd fix the bug. 👍

Reply:
wait, didnt i do that? do you want me just to remove the heading?

Reply:
oh and leave the link text as is? is that what you mean?

Reply:
Yes, the heading-related bit is the part I wanted to think about. The link fixes looked good, though.

Reply:
cool, will remove as soon as i escape all the meetings